ios/Constants.swift

```diff
@@ -23,6 +23,9 @@ let HKWorkoutTypeIdentifier = "HKWorkoutTypeIdentifier"
 let HKWorkoutRouteTypeIdentifier = "HKWorkoutRouteTypeIdentifier"
 let HKDataTypeIdentifierHeartbeatSeries = "HKDataTypeIdentifierHeartbeatSeries"
 
+let HKWorkoutActivityType_PROPERTY_NAME = "activityType"
+let HKWorkoutSessionLocationType_PROPERTY_NAME = "locationType"
```

Comment:
nit: Maybe we should use the same indentation here. 
```suggestion
let HKWorkoutActivityTypePropertyName = "activityType"
let HKWorkoutSessionLocationTypePropertyName = "locationType"
```

Reply:
Fixed, thanks!